### PR TITLE
Dont display 0 when claimed

### PR DESCRIPTION
**Pull request by bachstatter**
Screenshot 2022-12-02 at 1 23 43 pm

### CODE REVIEW

File: v2/components/RewardsItem/RewardsItem.tsx [hunk 1]
```diff
@@ -332,10 +332,14 @@ export const Rewards = () => {
         )}
         isLoading={isLoading}
         rewardBalance={
-          <Flex flexDirection="column">
-            <span>{formatNumber(rewardsData?.sUSDRewards.toNumber() || 0)} sUSD</span>
-            <span>{formatNumber(rewardsData?.snxRewards.toNumber() || 0)} SNX</span>
-          </Flex>
+          rewardsData?.hasClaimed ? (
+            <></>
```
Comment (jmzwar) [inline v2/components/RewardsItem/RewardsItem.tsx:336]: Can do && () here

Comment (jmzwar) [reply v2/components/RewardsItem/RewardsItem.tsx:336]: & double check the badge is vert aligned

Comment (bachstatter) [reply v2/components/RewardsItem/RewardsItem.tsx:336]: hmm I think the ui behave differently when you pass `undefined`. Looks like it does `-`.

Comment (bachstatter) [reply v2/components/RewardsItem/RewardsItem.tsx:336]: Probably better to change that instead.. You wanna clean it up? 🙏

Comment (jmzwar) [reply v2/components/RewardsItem/RewardsItem.tsx:336]: undefined is javascript existentialism

is hurts me spiritually :)

Comment (jmzwar) [reply v2/components/RewardsItem/RewardsItem.tsx:336]: I'll take a look

**Comment by bachstatter** [comment]
Latest commit will render:
image
See line 241

**Comment by jmzwar** [comment]
> Latest commit will render: image See line 241

Don't we expect that?

**Comment by bachstatter** [comment]
> > Latest commit will render: image See line 241
> 
> Don't we expect that?

No we want it to look like my screenshot:
Screenshot 2022-12-02 at 1 23 43 pm

Remove numbers and show "Claimed" badge

**Review by jmzwar — APPROVED** [review]

**Pull request merged**